content/news/2017-01-stats-post.md

```diff
@@ -0,0 +1,73 @@
+The current release has been out for almost three months now. Time to take a look at some statistics!
+
+At the time when this post was drafted, there have been a recorded 53953 downloads from the official OpenRA homepage. Not unsurprisingly, most downloaded packages are for the Windows platform (almost 75%!), with Linux being a distant second at 13% and OS X coming last with the remaining 12%. It is notable that the shares of both Linux and OS X are much larger than their respective share of the desktop market.
```

Comment:
It might be worth mentioning explicitly that we know many downloads are from downstream sources.  It might not be worth mentioning that this will systematically bias the downloads-vs-sysinfo stats.